modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala

```diff
@@ -171,6 +171,14 @@ object Databricks {
                 throw new IllegalStateException("Databricks Loader does not use EventsCopyToTempTable statement")
               case _: Statement.EventsCopyFromTempTable =>
                 throw new IllegalStateException("Databricks Loader does not use EventsCopyFromTempTable statement")
+              case Statement.VacuumEvents => sql"""
+                  OPTIMIZE ${Fragment.const0(qualify(EventsTable.MainName))}
+                  WHERE collector_tstamp_date >= current_timestamp() - INTERVAL ${tgt.eventsOptimizePeriod.toSeconds} second
+                  ZORDER BY collector_tstamp"""
+              case Statement.VacuumManifest => sql"""
+                  OPTIMIZE ${Fragment.const0(qualify(EventsTable.MainName))}
+                  WHERE collector_tstamp_date >= current_timestamp() - INTERVAL ${tgt.manifestOptimizePeriod.toSeconds} second
```

Comment:
I just realised... this assumes that the `collector_tstamp_date` exists!  I wonder if all open source users have added this column.  I cannot remember if we documented that this is a good column to manually add to the table.

On the other hand, if the `collector_tstamp_date` column does not exist then at least it won't crash the app.  And users without the column can disable the feature if they find they get errors in the logs.